File: src/main/java/com/github/dkuzmenkov17/todo/domain/ToDoDto.java
package com.github.dkuzmenkov17.todo.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Builder;
import lombok.Value;

@Value
@Builder
@JsonIgnoreProperties
public class ToDoDto {
    private Long id;
    private String title;
    private String description;
    private Status status;
}

File: src/main/kotlin/com/github/dkuzmenkov17/todo/dto/ToDoDto.kt
package com.github.dkuzmenkov17.todo.dto

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Calendar

@JsonInclude(JsonInclude.Include.NON_NULL)
data class ToDoDto(@JsonProperty("id") val id: Long,
                   @JsonProperty("title") val title: String = "Title $id",
                   @JsonProperty("is_done") val isDone: Boolean = false,
                   @JsonProperty("updated_at") val updatedAt: Calendar)